ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/integration/Record_Decision/103788_edit-deferred-decision.js

```diff
@@ -0,0 +1,343 @@
+    cy.deferredDecisionDate().should('contain.text', '10 October 2021')
+    // clicks on the record a decision button to submit
+    cy.recordThisDecision().click()
+    // recorded decision confirmation
+    cy.deferredProjectStateId().should('contain.text', 'Decision recorded')
+    cy.visit(projecList)
+    cy.projectStateId().should('contain.text', 'DEFERRED')
+})
+
+// Edit Deferred Path - Minister, Other
+it('TC04: J3 Edit a recorded decision Declined - Minister, Other', () => {
+    // Click on change your decision button 
+    cy.changeDecision().should('contain.text', 'Change your decision').click()
+     //select iniital decision
+    cy.deferredRadioBtn().click()
+    // clicks on the continue button
+    cy.continueBtn().click()
+    // selects regional director button
+    cy.get('[id="minister-radio"]').click()
+    // clicks on the continue button
+    cy.continueBtn().click()
+    cy.OtherCheckBox()
+    .invoke('attr', 'aria-expanded')
+    .then(ariaExpand => {
+        if (ariaExpand.includes(true)) {
+            cy.OtherCheckText().clear().type('This is the second test')
+        }
+        else {
+            // clicks on other box
+            cy.OtherCheckBox().click()
+            .then(()=> {
+            // clicks on the Give Reasons box
+            cy.OtherCheckText().clear().type('This is the first test')
+            })
+        }
+    })
+    // clicks on the continue button
+    cy.continueBtn().click()
+    // date entry
+    cy.recordDecisionDate(10, 10, 2021)
+    // clicks on the continue button
+    cy.continueBtn().click()
+    // Change condition
+    cy.deferredReasonChangeLink().click() 
+    cy.OtherCheckBox()
+    .invoke('attr', 'aria-expanded')
+    .then(ariaExpand => {
+        if (ariaExpand.includes(true)) {
+            cy.OtherCheckText().clear().type('This is the second test')
+        }
+        else {
+            // clicks on finance box
+            cy.OtherCheckBox().click()
+            .then(()=> {
+            // clicks on the Give Reasons box
+            cy.OtherCheckText().clear().type('This is the first test')
+            })
+        }
+    })
+    cy.continueBtn().click()
+    cy.continueBtn().click()
+    // preview answers before submit
+    cy.deferredDecision().should('contain.text', 'Deferred')
+    cy.deferredDecisionMadeBy().should('contain.text', 'Minister')
+    //cy.get('[id="deferred-reasons"]').should($el => expect($el.text().trim()).to.equal('Additional information needed:\n                    This is the second test\n                    Awaiting next ofsted report:\n                    This is the second test\n                    Performance concerns:\n                    This is the second test\n                    Other:\n                    This is the second test'))
```

Comment:
Rather than commenting this out and having to come back to it, if the order is not important you could do something like - 
```
var deferredReasons = cy.get('[id="deferred-reasons"]').trim
deferredReasons .should('contain.text', 'information needed:')
deferredReasons .should('contain.text', 'information details)
```
Etc etc
The other added benefit of this is it'll be easier to tell what has caused the test to fail when one of these assertions fails.

Reply:
You can put a cypress command in a variable.